applications/TrilinosApplication/custom_processes/trilinos_levelset_convection_process.h

```diff
@@ -257,10 +257,18 @@ class TrilinosLevelSetConvectionProcess
         // Generating the elements
         (BaseType::mpDistanceModelPart->Elements()).reserve(rBaseModelPart.NumberOfElements());
         for (auto it_elem = rBaseModelPart.ElementsBegin(); it_elem != rBaseModelPart.ElementsEnd(); ++it_elem){
-            Element::Pointer p_element = Kratos::make_intrusive< LevelSetConvectionElementSimplex < TDim, TDim+1 > >(
-                it_elem->Id(),
-                it_elem->pGetGeometry(),
-                it_elem->pGetProperties());
+            Element::Pointer p_element;
+            if (this->mIsAlgebraicStabilization){
+                p_element = Kratos::make_intrusive< LevelSetConvectionElementSimplexAlgebraicStabilization < TDim, TDim+1 > >(
+                    it_elem->Id(),
+                    it_elem->pGetGeometry(),
+                    it_elem->pGetProperties());
+            } else {
+                p_element = Kratos::make_intrusive< LevelSetConvectionElementSimplex < TDim, TDim+1 > >(
+                    it_elem->Id(),
+                    it_elem->pGetGeometry(),
+                    it_elem->pGetProperties());
+            }
```

Comment:
In the subsequent PR (in which the element type will be passed) we should better address this (outside the loop and with an auxiliary method or lambda).